tfx/components/example_gen/utils.py

```diff
@@ -393,9 +476,11 @@ def _create_matching_glob_and_regex(
   return split_glob_pattern, split_regex_pattern
 
 
-def _retrieve_latest_span_version(
+def _get_target_span_version(
     uri: Text,
-    split: example_gen_pb2.Input.Split) -> Tuple[Optional[int], Optional[int]]:
+    split: example_gen_pb2.Input.Split,
+    range_config: Optional[range_config_pb2.RangeConfig] = None
+) -> Tuple[Optional[int], Optional[int]]:
```

Comment:
Please update the docstring of this function too, it is not always `recent` span / version anymore.